10_cdisc_questions.md

```diff
@@ -0,0 +1,68 @@
+1. What is the main focus of Good Clinical Practices (GCP)? [comment]: <> (This is not appropriate for CDISC section, perhaps could be moved general programming section)
+    A. harmonized data collection
+    B. standard analysis practices
+    C. protection of subjects
+    D. standard monitoring practices
+    *** Answer: C. protection of subjects*** 
+
+ 2. Specify the variable role which focus on the observation (such as the name of a lab test) of a SDTM data set? 
+    A. Identifier
+    B. Topic
+    C. Timing
+    D. Qualifier
+    Answer: B. Topic
+
+ 3. Concomitant/Prior Medications (CM) is a component of which SDTM class?
+    A. Findings
+    B. Interventions
+    C. Events
+    D. Special Purpose
+    Answer: B. Interventions
+
+ 4. What is the primary purpose of programming validation? [comment]: <> (This is not appropriate for CDISC section, perhaps could be moved general programming section)

```

Comment:
28Sep22: We agreed to remove this question

Reply:
Can we add some of these to the MD file? I accepted the merge